benchmark/nixlbench/src/utils/cli_common.cpp

```
@@ -0,0 +1,196 @@
+splitFileNames(const std::string &value) {
+    std::vector<std::string> names;
+    std::stringstream input(value);
+    std::string name;
+    while (std::getline(input, name, ',')) {
+        names.push_back(name);
+    }
+    return names;
+}
+
+std::string
+formatSize(size_t bytes) {
```

Cppcheck (2.21.0)

[style] The function 'formatSize' is never used.

(unusedFunction)